integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/TccReferenceAnnotationFilter.java

```diff
@@ -79,7 +79,7 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
                 }
             }
         } catch (Throwable e) {
-            LOGGER.error("Tcc dubbo invokes service to register branch transaction exception:", e);
+            LOGGER.error("Tcc dubbo invokes service to register branch transaction exception: {}", e);
```

Comment:
LOGGER.error("Tcc dubbo invokes service to register branch transaction exception: {}", e.getMessage(),e);